src/jitlayers.h

```diff
@@ -334,6 +335,8 @@ class JuliaOJIT {
     TargetMachine &getTargetMachine();
     const Triple& getTargetTriple() const;
     size_t getTotalBytes() const;
+
+    JITDebugInfoRegistry &getDebugInfoRegistry();
```

Comment:
```suggestion
    JITDebugInfoRegistry &getDebugInfoRegistry() const;
```

Reply:
I don't think we can mark this method as const, since it returns a non-const reference to the debug info registry? And that reference has to be non-const because debuginfo will change it, but we protect it so thread safety shouldn't be an issue at this point.

Reply:
ah, okay. I suppose marking `const this` would force it to require the return type to be `const JITDebugInfoRegistry&`?